core/src/main/scala/kafka/log/Log.scala

```diff
@@ -277,6 +278,24 @@ class Log(@volatile var dir: File,
     },
     tags)
 
+  newGauge("LogSegments",
+    new Gauge[util.List[String]] {
+      def value = {
+        val list = logSegments.toSeq.map { seg =>
+          s"baseOffset=${seg.baseOffset}, created=${seg.createdMs}, logSize=${seg.size}, indexSize=${seg.indexSize}"
```

Comment:
There is a comment on your previous PR about using a different delimiter. Can you address that.

Reply:
Would `|` be OK?

Reply:
Perhaps a ';' (semi-colon)?
```suggestion
          s"baseOffset=${seg.baseOffset}; created=${seg.createdMs}; logSize=${seg.size}; indexSize=${seg.indexSize}"
```

Reply:
Seems better. Updated.

Reply:
Seems better. Updated.